docs/explanations/how-workflows-work.md

```diff
@@ -23,17 +24,17 @@ Before a workflow starts, DBOS records its input.
 Each time a workflow executes a [transaction](../tutorials/transaction-tutorial) or [communicator](../tutorials/communicator-tutorial), DBOS records its output (or the exception it threw, if any).
 When a workflow finishes, DBOS records its output.
 
-If a DBOS server crashes and restarts, it uses the information saved in the database to resume all unfinished workflows from where they left off.
-First, it finds all unfinished workflows: those with a recorded input, but no recorded output.
-Then, it restarts every unfinished workflow from the beginning, using its saved inputs.
-While re-executing an unfinished workflow, it checks before every function execution if the function has an output stored in the database, meaning it previously completed.
+If a DBOS server crashes and restarts, it uses the information saved in the database to resume all pending workflows from where they left off.
+First, it finds all pending workflows: those with a recorded input, but no recorded output.
+Then, it restarts every pending workflow from the beginning, using its saved inputs.
+While re-executing an pending workflow, it checks before every function execution if the function has an output stored in the database, meaning it previously completed.
 If it finds a saved output, it skips re-executing that function and instead uses the saved output.
 When the workflow gets to the first function that does not have a saved output and hence _didn't_ previously complete, it executes normally, thus "resuming from where it left off."
-Let's look at how this procedure gets us all three of our guarantees.
+Let's look at how this procedure obtains all three guarantees above.
 
 1.  Any interrupted workflow is re-executed until it completes, so workflows always run to completion.
```

Comment:
Mention again that we always restart: "Any interrupted workflow is re-started and re-executed..."